test/test-with-server.js

```diff
@@ -235,6 +236,21 @@ describe("follow-redirects", function () {
       });
   });
 
+  it("should callback setTimeout on the returned stream", function () {
+    app.get("/redirect", redirectsTo("/timeout"));
+    app.get("/timeout", delaysJson(5000, { time: "out" }));
```

Comment:
we should probably have a test where each one is delayed by 1s or so

Reply:
+1